docs/release-notes/7.17.asciidoc

```diff
@@ -2,14 +2,25 @@
 == 7.17
 
 [discrete]
-[[release-notes-7.17.1]]
-=== 7.17.1
+[[release-notes-7.17.2]]
+=== 7.17.2
 
 [discrete]
-[[bug-fixes-7.17.1]]
+[[bug-fixes-7.17.2]]
 ==== Bug fixes and enhancements
+* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted under certain circumstances.
+* The bug that prevented Endpoint Security from being run on Windows 8.1 and Server 2012 R2 has been fixed. With this change, the primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc (https://github.com/elastic/endpoint/issues/15[#15]).
+* Updates the minimum role permissions needed to import rules with actions. After this change, roles must have at least `Read` privileges for the `Actions and Connectors` feature to import rules with actions ({pull}126203[#126203]).
 
-There are no user-facing changes in the 7.17.1 release.
+[discrete]
+[[release-notes-7.17.1]]
+=== 7.17.1
+
+[discrete]
+[[known-issue-7.17.1]]
+==== Known issues
+* An {endpoint-sec} integration bug will prevent benign Windows files from being deleted under certain circumstances.
+* Endpoint Security cannot be run on Windows 8.1 and Server 2012 R2 (https://github.com/elastic/endpoint/issues/15[#15]).
```

Comment:
Endpoint Security 7.17.1 works on all Windows variants. It's 8.1.1 that doesn't work on those two Windows versions.

```suggestion
```

Reply:
Removing since this isn't a bug with 7.17.1. It's being added to 7.17.2 as a preventative fix.